content/en/Platform Deep Dive/Collaboration/Organization/your-contract.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Your Cobalt Contract"
+linkTitle: "Your Contract"
+weight: 30
+description: >
+  Learn how your Cobalt contract works.
+---
+
+{{% pageinfo %}}
+This page provides an overview of terms related to your organization's contract with Cobalt. For detailed information, read our [Terms & Conditions](https://www.cobalt.io/terms).
+{{% /pageinfo %}}
+
+Learn what to expect when your organization's contract is [active](#when-your-contract-is-active) or [expires](#when-your-contract-expires).
+
+## When Your Contract is Active
+
+Once you've signed a contract with Cobalt and purchased [credits](/platform-deep-dive/credits/), you get access to our Pentest as a Service (PtaaS) platform. You can start using your credits from the start date of your contract. Credits are not available prior to that date.
+
+When your organization's contract is active, you have full access to the platform, as determined by your [user role](/platform-deep-dive/collaboration/organization/user-roles/). If your role allows, you can:
+
+- Launch [pentests](/platform-deep-dive/pentests/) to test your [assets](/platform-deep-dive/assets/).
+- Remediate [findings](/platform-deep-dive/pentests/findings/) and [submit them for retest](/platform-deep-dive/pentests/findings/remediate-findings/#submit-a-finding-for-retest).
+- View and download [pentest reports](/platform-deep-dive/pentests/reports/) once they're ready.
+- Collaborate on pentests.
+- Use other platform features, as determined by your {{% ptaas-tier %}}.
+
+{{< alert title="Tip" color="primary" >}}
+To renew your contract, reach out to your Customer Success Manager or support@cobalt.io.
+{{< /alert >}}
+
+## When Your Contract Expires
+
+When your organization's contract is about to expire, we notify you in advance. Organization users get notifications reminding them to renew the contract.
+
+- Be sure to start your pentests at least 30 days before your contract expires, if you have credits remaining.
+  - The standard testing period is 14 days. It may vary depending on the pentest scope and other factors.
+- [Submit your findings for retest](/platform-deep-dive/pentests/findings/remediate-findings/#submit-a-finding-for-retest) at least 10 days before your contract expires. Learn more about the [free retesting duration](/platform-deep-dive/pentests/findings/remediate-findings/#free-retesting-duration).
+- [Download the data you need](#download-your-organizations-pentest-data) before we revoke your access to the platform.
+
+After the contract expiration date, a **15-day grace period** starts.
+
+- Within the grace period, you still have access to the platform. Be sure to [download the data you need](#download-your-organizations-pentest-data) within this time frame.
+- Your remaining [credits](/platform-deep-dive/credits/) expire.
+- You can no longer submit pentests or retests for review.
```

Comment:
pentests are submitted for review, but findings are submitted as ready for retest. Should we change "You can longer submit pentests or retests for review." to "You can no longer submit pentests for review or findings for retest."